modules/apps/object/object-test/src/testFunctional/tests/ObjectFields.testcase

```diff
@@ -2024,6 +2024,54 @@ definition {
 		}
 	}
 
+	@description = "LPS-170122 - Verify that is not possible to edit the value of field when is read-only property is conditional"
+	@priority = 5
+	test CannotEditConditionalReadOnlyFieldWhenConditionIsTrue {
+		task ("Given an object definition") {
+			ObjectAdmin.addObjectViaAPI(
+				labelName = "Custom Object",
+				objectName = "CustomObject",
+				pluralLabelName = "Custom Objects");
+		}
+
+		task ("And given a long text field with the the read-only property set as conditional with an expression") {
+			ObjectAdmin.addObjectFieldViaAPI(
+				fieldBusinessType = "LongText",
+				fieldLabelName = "Custom Field",
+				fieldName = "customField",
+				fieldType = "Clob",
+				isReadOnly = "conditional",
+				isRequired = "false",
+				objectName = "CustomObject",
+				readOnlyExpression = "customField == 'Entry Test'");
+
+			ObjectAdmin.publishObjectViaAPI(objectName = "CustomObject");
+		}
+
+		task ("And given an object entry that makes the conditional expression true") {
+			Navigator.openURL();
+
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+			ObjectAdmin.addObjectSingleFieldEntryViaAPI(
+				fieldName = "customField",
+				objectName = "CustomObject",
+				value = "Entry Test");
+		}
+
+		task ("When the entry is being edited") {
+			ObjectAdmin.goToCustomObject(objectName = "CustomObject");
+
+			ObjectPortlet.viewEntryDetails(entry = "Entry Test");
+		}
+
+		task ("Then it's not possible to edit the field's value") {
+			AssertNotEditable(
+				key_optionName = "Entry Test",
+				locator1 = "ObjectAdmin#VIEW_FIELD_LONG_TEXT_DISABLED");
```

Comment:
@thaiscabrals have you checked if any of our existing path works before creating a new one?

Reply:
Yes, I checked and there is no path available to check if it is not possible to edit a long text.

Reply:
@thaiscabrals is this path unique for long text or does it apply to any read-only field? I ask because the path specifies long text and I wanted to check if we should rename it to something generic.

Reply:
@timpak This path is only for Long Text.
We have the path `VIEW_FIELD_VALUE_DISABLED` but it does not work for long text.